code/game/machinery/adv_med.dm

```diff
@@ -198,6 +198,21 @@
 	update_icon()
 	set_light(0)
 
+/obj/machinery/bodyscanner/emag(mob/user)
+	if(!emagged)
+		if(user)
+			to_chat(user, "<span class='warning'>You disable the X-ray dosage limiter on \the [src].</span>")
+		to_chat(user, "<span class='notice'>\The [src] emits an ominous hum.</span>")
```

Comment:
What's the point of using `if(user)` and then not indenting this line? Indent this line and line 211 too or drop the if(user) entirely (I actually have no idea if user can be null, it doesn't seem to be the case)